cupy/indexing/generate.py

```diff
@@ -244,7 +248,105 @@ def ix_(*args):
         out.append(new)
     return tuple(out)
 
-# TODO(okuta): Implement ravel_multi_index
+
+def ravel_multi_index(multi_index, dims, mode='raise', order='C'):
```

Comment:
The default mode shouldn't be raise but `wrap` to avoid hidden device synchronization.

https://docs-cupy.chainer.org/en/stable/reference/difference.html